examples/skip_plot_functionality.py

```diff
@@ -0,0 +1,52 @@
+"""
+=====================
+Adding extra features
```

Comment:
I think this title needs changing.

Reply:
Does "Extending functionality from sunpy" sound better?

Reply:
Yeah that works.